intellij-toml/core/src/test/kotlin/org/toml/ide/inspections/TomlPropertyRedefinitionInspectionTest.kt

```diff
@@ -0,0 +1,71 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.toml.ide.inspections
+
+class TomlPropertyRedefinitionInspectionTest : TomlInspectionsTestBase(TomlPropertyRedefinitionInspection::class) {
+    fun `test in keys`() = checkByText("""
+        foo = "bar"
+        <error descr="Property redefinition is not allowed">foo</error> = "baz"
```

Comment:
Also, I think the error should appear for both keys, like in json:
![image](https://user-images.githubusercontent.com/3221931/127482242-1a37829b-2984-4872-957d-59c4856a9eef.png)